app/src/main/java/toasty/messageinabottle/LoginActivity.java

```diff
@@ -310,24 +314,32 @@ private void addEmailsToAutoComplete(List<String> emailAddressCollection) {
         @Override
         protected Boolean doInBackground(Void... params) {
             // TODO: attempt authentication against a network service.
-
+            final String tag="Live Login Activity";
             try {
                 // Simulate network access.
                 Thread.sleep(2000);
             } catch (InterruptedException e) {
                 return false;
             }
 
-            for (String credential : DUMMY_CREDENTIALS) {
-                String[] pieces = credential.split(":");
-                if (pieces[0].equals(mEmail)) {
-                    // Account exists, return true if the password matches.
-                    return pieces[1].equals(mPassword);
+            try {
```

Comment:
The strategy of trying to create an account or doing login seems like it opens the door to error. We should add a button called register instead.
i.e. imagine the scenario wherein you misspell your email and suddenly have a new account